modules/signing_worker/files/requirements.txt

```diff
@@ -0,0 +1,5 @@
+-e git+https://github.com/mozilla-releng/scriptworker-scripts#egg=mozbuild&subdirectory=vendored/mozbuild
+-e git+https://github.com/mozilla-releng/scriptworker-scripts#egg=scriptworker_client&subdirectory=scriptworker_client
+-e git+https://github.com/mozilla-releng/scriptworker-scripts#egg=iscript&subdirectory=iscript
```

Comment:
Now that the mono-repo is populated with the other workers there are `production` and `production-<worker>` branches for them, while we're on master for iscript and deps. While we don't need to merge iscript to a branch to deploy I'm wondering if we can be consistent somehow ? Maybe a `production-iscript` branch that we use here, and some docs which point out how iscript differs from GCP-workers ?

Reply:
I'm happy with whatever branch I'm pointed to, so I'm assuming this is more a question for @escapewindow

Reply:
I don't love the `production*` branches and plan on obsoleting them (by porting scriptworker-scripts to taskgraph + CoT, and using relpro instead of force-pushes to ship docker images). However, these requirements.txt lines are candidates for future fixes as well: once we have wheel generation in the monorepo, we should use those. So I'm -0 on this :)

Reply:
OK, lets leave it as is.